Gemfile

```diff
@@ -1,6 +1,7 @@
+# frozen_string_literal: true
+
```

Comment:
If not necessary, (here it is not, I think) I would not use the `# frozen_string_literal` directive.
```suggestion
```